specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/blueprintDefinition.json

```diff
@@ -1205,9 +1205,21 @@
             "type": "string",
             "description": "Name of peer artifact."
           }
+        },
+        "tags": {
```

Comment:
While not a strict rule, this is best done in an incremented api version. If a user with an updated client populates this value, and then a user with a legacy client does a GET/MERGE/PUT to modify the resource, the property would be wiped out.

Reply:
This was a risk we were aware of but deemed the risk low enough to not require a version bump. If we weren't in preview we wouldn't take any such risk, even low, but in this case we see it as highly unlikely two users would be modifying the same blueprint with two separate SDK releases. If possible we'd like to push this one through to the existing version.

Reply:
Ok